src/DynamoCoreWpf/TestInfrastructure/CodeBlockNodeMutator.cs

```diff
@@ -61,7 +61,7 @@ public override bool RunTest(NodeModel node, EngineController engine, StreamWrit
 
                 DynamoViewModel.ExecuteCommand(runCancel);
             }));
-            Thread.Sleep(100);
+            Thread.Sleep(0);
```

Comment:
Can you explain why this `Thread.Sleep` call is present? There are different, guaranteed mechanisms for flushing the `Dispatcher`. For example, see `NodeViewCustomizationTests`.